src/main/java/testUI/elements/Scrolling.java

```diff
@@ -53,19 +54,28 @@ public UIElement customSwipeUp(int PixelGap, int numberOfSwipes) {
         try {
             if (!Configuration.automationType.equals(Configuration.DESKTOP_PLATFORM)) {
                 for (int i = 0; i < numberOfSwipes; i++) {
-                    TouchAction action = new TouchAction(getDriver());
                     int startY = 500;
                     PixelGap = abs(PixelGap);
                     int endY = 500 - PixelGap;
                     if (endY < 0) {
                         endY = 100;
                         startY = endY + PixelGap;
                     }
-                    action.press(
-                            PointOption.point(40, startY)
-                    ).waitAction(
-                            WaitOptions.waitOptions(Duration.ofMillis(300))
-                    ).moveTo(PointOption.point(40, endY)).release().perform();
+
+                    Dimension size = getDriver().manage().window().getSize();
+                    int endX = (int) (size.width * 0.10);
+
+                    PointerInput finger = new PointerInput(PointerInput.Kind.TOUCH, "finger");
+                    Sequence sequence = new Sequence(finger, 1);
+                    sequence.addAction(finger.createPointerMove(ofMillis(0),
+                            PointerInput.Origin.viewport(), endX, startY));
+                    sequence.addAction(finger.createPointerDown(PointerInput.MouseButton.MIDDLE.asArg()));
+                    sequence.addAction(new Pause(finger, ofMillis(600)));
```

Comment:
Any reason why exactly 600ms?

Reply:
Not adding any time can induce a misbehaviour, the exact 600ms is just a random number that often works fine